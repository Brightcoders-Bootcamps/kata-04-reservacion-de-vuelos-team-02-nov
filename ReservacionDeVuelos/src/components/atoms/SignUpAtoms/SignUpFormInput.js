import React from 'react';
import {Text, TextInput} from 'react-native';
import {styles} from '../../../styles/SignUpStyles';
const SignUpFormInput = (props) => {
  const {inputName} = props;

  return (
    <>
      <Text style={styles.formText}>{inputName}</Text>
      <TextInput
        style={styles.textInput}
        secureTextEntry={inputName === 'Password *' ? true : false}
      />
      {inputName === 'Password *' && (
        <Text style={styles.characters}>
          Use 8 or more characters with a mix of letters, numbers, and sumbols
        </Text>
      )}
    </>
  );
};

export default SignUpFormInput;
