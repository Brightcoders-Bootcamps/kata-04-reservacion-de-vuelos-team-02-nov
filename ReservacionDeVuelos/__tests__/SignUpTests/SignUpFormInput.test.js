import React from 'react';
import SignUpFormInput from '../../src/components/atoms/SignUpAtoms/SignUpFormInput';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<SignUpFormInput />).toJSON();
  expect(tree).toMatchSnapshot();
});