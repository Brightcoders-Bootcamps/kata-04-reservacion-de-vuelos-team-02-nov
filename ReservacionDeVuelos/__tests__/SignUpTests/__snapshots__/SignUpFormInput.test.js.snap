// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <Text
    style={
      Object {
        "fontSize": 16,
        "marginBottom": 10,
      }
    }
  />,
  <TextInput
    allowFontScaling={true}
    rejectResponderTermination={true}
    secureTextEntry={false}
    style={
      Object {
        "borderColor": "gray",
        "borderWidth": 1,
        "height": 40,
        "marginBottom": 10,
      }
    }
    underlineColorAndroid="transparent"
  />,
]
`;
